Small changes for bookkeeping

fabianhervaspeters opened the pull request:
## Summary

I wanted to do this in two different pull requests but did not manage. This PR makes a few small changes in the modules for bookkeeping and adds config files to the example folder in cfis_simu which allowed me to run the simulated images through shapepipe. 

## Reviewer Checklist

> Reviewers should tick the following boxes before approving and merging the PR.

- [x] The PR targets the `develop` branch
- [x] The PR is assigned to the developer
- [x] The PR has appropriate labels
- [x] The PR is included in appropriate projects and/or milestones
- [x] The PR includes a clear description of the proposed changes
- [ ] If the PR addresses an open issue the description includes "closes #<ISSUE NUMBER>"
- [x] The code and documentation style match the current standards
- [x] Documentation has been added/updated consistently with the code
- [ ] All CI tests are passing
- [ ] API docs have been built and checked at least once (if relevant)
- [ ] All changed files have been checked and comments provided to the developer
- [ ] All of the reviewer's comments have been satisfactorily addressed by the developer

fabianhervaspeters committed:
get images file handling bug

martinkilbinger committed:
Merge remote-tracking branch 'fabian/fabian_simu' into fabian_simu

fabianhervaspeters committed:
prefix find_exp_runner

fabianhervaspeters committed:
fix uint32, still error

fabianhervaspeters committed:
Merge branch 'CosmoStat:develop' into fabian_simu

fabianhervaspeters committed:
new config files simu

fabianhervaspeters committed:
Merge branch 'fabian_simu' of github.com:fabianhervaspeters/shapepipe into fabian_simu

fabianhervaspeters committed:
Delete .DS_Store

fabianhervaspeters committed:
Delete ._.DS_Store

martinkilbinger commented on example/cfis_simu/config_tile.mask:
```diff
@@ -0,0 +1,90 @@
+# Mask module config file for tiles
```
Is this config file being used?

martinkilbinger commented on example/cfis_simu/job_sp_simu.bash:
```diff
@@ -0,0 +1,462 @@
+#!/usr/bin/env bash
```
should be in the `scripts/sh` directory.

Can this be merged with `job_sp.bash`?

martinkilbinger commented on example/cfis_simu/mask_default/MEGAPRIME_star_i_13.8.reg:
```diff
@@ -0,0 +1,24 @@
+-11.5 68
```
I think some of those files used for masking (e.g. Messier & NGC objects) are not used.

martinkilbinger commented on shapepipe/modules/find_exposures_package/find_exposures.py:
```diff
@@ -28,12 +28,13 @@ class FindExposures():
     """
 
-    def __init__(self, img_tile_path, output_path, w_log, colnum):
+    def __init__(self, img_tile_path, output_path, w_log, colnum, prefix):
```
Update class doc string

martinkilbinger commented on shapepipe/modules/find_exposures_runner.py:
```diff
@@ -35,12 +35,15 @@ def find_exposures_runner(
     colnum = config.getint(module_config_sec, 'COLNUM')
 
+    # Give the prefix of exposures
+    exp_prefix = config.get(module_config_sec, 'EXP_PREFIX')
```
Update doc string in __init__.py

martinkilbinger commented on shapepipe/modules/mask_package/mask.py:
```diff
@@ -1179,6 +1179,7 @@ def _build_final_mask(
             )
             external_flag.open()
             if final_mask is not None:
+                final_mask=final_mask.astype(np.int16, copy=False)
```
spaces around '='.

martinkilbinger commented:
Write a brief summary of the PR, and set the assignment and label.

fabianhervaspeters committed:
Delete config_tile.mask

As Martin suggested this file is not needed when running the simulation as config_tile.mask_simu is used

martinkilbinger committed:
Merge remote-tracking branch 'fabian/fabian_simu' into fabian_simu

martinkilbinger committed:
Merge remote-tracking branch 'upstream/develop' into fabian_simu

martinkilbinger committed:
added spaces

martinkilbinger committed:
Merge remote-tracking branch 'upstream/develop' into fabian_simu

fabianhervaspeters committed:
added doctsring

fabianhervaspeters committed:
added docstring

fabianhervaspeters committed:
add space

fabianhervaspeters committed:
Merge branch 'fabian_simu' of github.com:fabianhervaspeters/shapepipe into fabian_simu

martinkilbinger reviewed (approved):
OK

martinkilbinger commented:
![image](https://github.com/CosmoStat/shapepipe/assets/4549196/f0b78570-6ed1-4a75-84f2-2936c3bff957)
